src/software/simulated_tests/simulated_test_fixture.h

```diff
@@ -190,6 +192,11 @@ class SimulatedTestFixture : public ::testing::Test
     // time passes at the same speed a real life time
     bool run_simulation_in_realtime;
 
+    double total_tick_duration;
+    double max_tick_duration;
+    double min_tick_duration;
+    unsigned int tick_count;
```

Comment:
Leaving some comments about the variables might be helpful.

Reply:
added comments